<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavData</class>
 <widget class="QWidget" name="NavData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLabel" name="l_COG">
     <property name="text">
      <string>Route Fond :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="l_SOG">
     <property name="text">
      <string>Vitesse Fond :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="l_dtPC">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="l_Position">
     <property name="text">
      <string>Position :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l_DateHeureGPS">
     <property name="text">
      <string>Données non reçues</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="3">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
